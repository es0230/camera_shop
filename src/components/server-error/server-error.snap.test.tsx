/**
 * @jest-environment jsdom
 */

import { render } from '@testing-library/react';
import ServerError from './server-error';

describe('testing ServerError component', () => {
  it('should render correctly', () => {
    const { container } = render(<ServerError />);

    expect(container).toMatchSnapshot();
  });
});
