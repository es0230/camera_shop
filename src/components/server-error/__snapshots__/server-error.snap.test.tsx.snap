// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing ServerError component should render correctly 1`] = `
<div>
  <p
    style="font-size: 72px; text-align: center; margin-top: 100px; line-height: 100px;"
  >
    Ошибка на сервере, вернитесь позднее
  </p>
</div>
`;
